### PR TITLE
Feat: Rulespage

### PR DESCRIPTION
**Pull request by yashpatel1797**
Rules page added on quiz app. 
image

### CODE REVIEW

**Review by zabihshaik — APPROVED** [review]
Looks Good to merge.

File: style/rules.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  border-radius: 1rem;
+}
+
+.rules-info {
+  display: flex;
+  align-items: center;
+}
+p > span {
+  margin-right: 1rem;
+}
```
Comment (sharmayatendra) [inline style/rules.css]: use class for targetting rather than an element.

**Review by sharmayatendra — APPROVED** [review]
can look upon suggestion. else looks good to merge

File: style/rules.css (hunk 1 above)
Comment (yashpatel1797) [reply style/rules.css]: will make changes

**Pull request merged**